examples/config_dc2_test_med.yaml

```diff
@@ -0,0 +1,85 @@
+# This is an example of the campaign level template specification file for dc2-test-med type campaigns
+
+# Import the Step templates for DC2
+- import: "${CM_CONFIGS}/config_dc2_steps.yaml"
+# Now build the Campaign Template for DC2 test med
+- SpecBlock:
+      name: d2c_campaign
+      # Define collections associated with the campaign
+      collections:
+          root: 'cm/dc2_test_med'
+      # Define the steps, their connections and override parameters as needed
+      child_config:
+          step1:
+              spec_block: dc2_step1
+              child_config:
+                  base_query: "instrument='LSSTCam-imSim' and skymap='DC2' and tract in (3828, 3829)"
+                  split_min_groups: 3
+          step2:
+              spec_block: dc2_step2
+              prerequisites: ['step1']
+              child_config:
+                  base_query: "instrument='LSSTCam-imSim' and skymap='DC2' and tract in (3828, 3829)"
+          step3:
```

Comment:
This file all looks pretty familiar. I think it's decently straightforward. It's not clear what makes some things fall under `child_config`. I'm getting the impression that all the `spec_block` s here are linked `config_dc2_steps.yaml` which seems good. What does the linking/how does that work?

Reply:
child_config is pretty much anything that you need to know how to make the child nodes.  

the linking happens when you make the child nodes, as part of that process you look up which spec blocks to use as templates for each child node.    You can see an example of that around here: https://github.com/lsst-dm/cm-service/blob/05a9a04824801e72b1f20607406d3a49d0db9d4a/src/lsst/cmservice/handlers/element_handler.py#L144